### ISSUE TITLE
[Bug]: wrong tag format for new entries 

### ISSUE THREAD

**Issue by FriedrichStein** [RafaelGB/obsidian-db-folder#777]
### Contact Details

j.friedrich-stein@gmx.de

### What happened?

When adding a file to a DB that is based on tag search, the new created file gets a "##tag" instead auf "#tag" and doesn't show up in the DB. 

### What platform were you using?

Desktop

### Version of the plugin

3.2.4

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by FriedrichStein** [comment]
Captured it. 
Thereby, I got another bug as well. Sometimes adding a line will also add seemingly radom files to the list(sometimes a lot of them). > (In the gif. "Universelle Gasgleichung" doens't belong in the DB and got added.)
I haven't narrowed it down yet, but it only happend with tags for me. 
They are gone, when reopening the DB.

![Jan-25-2023 13-03-39](https://user-images.githubusercontent.com/53973194/214559017-11f23ecd-ed02-4d9f-ac7c-0c76435c3873.gif)


Just saw the new auto update option. Thats so nice!! Thanks!
